stl/inc/yvals.h

```diff
@@ -208,6 +208,21 @@ _STL_DISABLE_CLANG_WARNINGS
 #define _STL_INTERNAL_STATIC_ASSERT(...)
 #endif // _ENABLE_STL_INTERNAL_CHECK
 
+#ifndef _ENABLE_ATOMIC_REF_ALIGNMENT_CHECK

```

Comment:
Do you have a use case for this being different than `_DEBUG` ? OK to leave as is, just curious.

Reply:
Pinpoint tearing in release, that never happens in debug due to timings and debug allocations